### DISCUSSION TITLE
unable to download  on certain website, return Exit code: 1

### DISCUSSION THREAD

**Discussion by tempexxx** [aclap-dev/video-downloadhelper#375] [Help and Q&A]
### Your question

Hello and greeting.

I would like to seek an advice or solution regarding my issue.
All this while I was able to download videos from a certain website. However, suddenly starting from yesterday VDH always return error exit code 1 as follows. 

Exit code: 1
moz-extension://8cb07f1d-0e26-4fc6-a13c-28f3b6fa1ca6/background/main.js:1
receive@moz-extension://8cb07f1d-0e26-4fc6-a13c-28f3b6fa1ca6/background/main.js:1:13157
nd</callCatchAppNotFound/</<@moz-extension://8cb07f1d-0e26-4fc6-a13c-28f3b6fa1ca6/background/main.js:1:42626

video resource: http$://missav.com/en/nima-028-uncensored-leak

works fine on other sites thou. 
Thank you in advanced. Your help is greatly appreciated.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:109.0) Gecko/20100101 Firefox/115.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "win7",
    "arch": "x86_64",
    "node": "10"
  },
  "home": "C:\\Users\\xxx",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Taomulong** [comment]
Doesn't work on Windows 10/11 in UK either. Might be due to the site, changed the way the video is delivered, or some annoying DRM kinda thing. Thankfully you can search "NIMA-028" and find plenty other options.

**Comment by Mattmjs** [comment]
> video resource: http$://missav.com/en/nima-028-uncensored-leak

I tried this one and it worked in 8.2.2.8 stable. If it doesn't work try the beta extension. I tried something from the same site
yesterday and it had an error. It seems to be intermittent.

**Reply by xecus922** [reply]
This is still happening for me as well on 8.2.2.8 stable. Was downloading fine yesterday now I'm getting the same 'Exit Code:1' errors on every download attempt. 

CoApp is fully updated.

Beta download link on the VDH site for Firefox at https://www.downloadhelper.net/w/Beta-versions seems to be down so I can't try that, just keeps trying to load the download forever. Firefox is fully updated. 

I hope you guys figure this one out for us, this is one of the highest quality Japanese sites available and a lot of the videos can only be found there now.I would know, I've done easily over a thousand Japanese video searches and commonly find this is the last site with the video I'm looking for left.

**Comment by GrampaWildWilly** [comment]
You need to read #519 carefully.  All of it.  Including any posts that GitHub might hide behind a comment like , "Show x previous replies."

If you want a reliable place to find the beta versions of VDH (& all versions of the CoApp for that matter), click the Wiki link at the top of any page of this forum.  That is a user manual.  Get in the habit of referring to it often.

**Reply by xecus922** [reply]
I'm aware of this post but I'm busy lately and would rather not deal with so many headaches and extra work to download every single video, 15 to 20 retries for one video as you admitted yourself. That's pretty ridiculous and I need far more than just one video. And VDH has worked very smoothly on this site for years up until very recently. 

The beta version (for Firefox) on the Wiki page that you're referring to as "reliable" uses the same exact link as the one on the VDH website and both are broken.

**Comment by xecus922** [comment]
Looks like it's fixed (I hope) after this latest update to stable that just happened today. Version 9.0.1.13 (stable) on Firefox.

Good job VDH team, thank you.

**Reply by paulrouget** [reply]
Thanks! Let me know if it breaks again… sorry for the trouble.

**Reply by Helen-ly5** [reply]
I'd like to know the URL of the stable and if it has an app to download?
